app/views/exercises/_form.html.erb

```diff
@@ -66,6 +66,7 @@
           <div class="col-md-8">
             <div class="field form-group">
               <%= f.editor :expectations_yaml, :yaml %>
+              <small><%= t('what_exactly_are_expectations_html') %></small>
```

Comment:
Have you checked this actually works? Since the translation has html code embedded, I expect Rails to escape that string.

You should use ```raw t``` instead of just `t` here